azure-pipelines.yml

```diff
@@ -1,166 +1,38 @@
-# Pipelines are made of one or more jobs and may include resources and variables.
-# Jobs are made of one or more steps plus some job-specific data. Steps can be
-# tasks, scripts, or references to external templates.
+pool:
+  vmImage: $(imageName)
 
 jobs:
   - job: Linux
     pool:
-      vmImage: 'ubuntu-16.04'
+      vmImage: ubuntu-latest
     strategy:
-      maxParallel: 2
       matrix:
-        py36:
-          PY_VERSION: 3.6
-        py27:
-          PY_VERSION: 2.7
+        py38:
+          PYTHON_VERSION: 3.8
+        py39:
+          PYTHON_VERSION: 3.8
```

Comment:
```suggestion
          PYTHON_VERSION: 3.9
```

Reply:
Thank you. please do not feel offended if i squash this in the end. Part of this exercise is trying to be "careful" in not destroying things.